include/mg_procedure.h

```diff
@@ -1581,6 +1587,10 @@ enum mgp_error mgp_func_result_set_error_msg(struct mgp_func_result *result, con
 /// mgp_func_result.
 enum mgp_error mgp_func_result_set_value(struct mgp_func_result *result, struct mgp_value *value,
                                          struct mgp_memory *memory);
+
+// Log a message on certain level with spdlog
```

Comment:
```suggestion
/// Log a message on a certain level.
```

Is `spdlog` an important factor here? As it is kind of an internal thing, I wouldn't mention this on the public docs, unless it is important. E.g.: if the format strings and parameter would be exposed, then it is important to know what is supported. As only a single string is passed into the function, I think it is irrelevant which logger do we use.

Reply:
Now that you say, it is not important 👍🏻 So as you say it is internal thing and I will remove it 👍🏻

Reply:
You missed the "a" article before "certain level".